### ISSUE TITLE
mysql connection error

### ISSUE THREAD

**Issue by florinbaci** [dbcodeio/public#169]
i try to connect to my linux server mysql db from my vs codium on my chromebook and i get this error.

I'm able to connect via ssh through terminal to the server!

Error:
connect EHOSTUNREACH 100.103.245.89:3306

---
VSCode Version: 1.96.2
Extension Version: 1.5.8
OS: linux
Arch: x64

**Comment by mikeburgh** [comment]
Thanks for raising.

In the host field, do you have that :3306 entered? It should just be the ip address.

**Comment by mikeburgh** [comment]
Closing this, re open if you are still having issues

**Comment by florinbaci** [comment]
I have solved this in the end using chat gpt, I still don't know what caused the problem bu as far as I my understanding goes I've switched the port on 3307 and it used the "connection string" to send first the info to my localhost and then through 3307 to my server if this makes any sense for an experienced person...

**Comment by mikeburgh** [comment]
Glad to hear you figured it out. Just confirming though, that screenshot is not from this extension

**Comment by florinbaci** [comment]
I've tried more or less what I could get my hands on from extensions and everywhere I've stumbled on the same problem, I thought that this my be useful in some circumstances, since I thought they may work in similar ways. 

I may be very wrong... ;)

But thanks anyways for replies!

**Comment by mikeburgh** [comment]
Ahh okay, now I follow.. Did you get it working in DBCode as well ?

**Comment by florinbaci** [comment]
nope, finally i have found an easier way to install extensions directly into the server, so probably will work with your extension as well, connect to server with "remote connection" extension and then extensions could be installed when you open a folder on the server!

maybe I'll your extension one day... thanks!
